composer.json

```diff
@@ -15,7 +15,7 @@
     "psr-4": { "drunomics\\ServiceUtils\\": "src/" }
   },
   "require": {
-    "drupal/core": "8.9.*"
+    "drupal/core": "^8.9 || ^9"
```

Comment:
read the README about branching strategy & releases. let's keep following that. #nw

Reply:
@fago so do we need a new branch?

> Major version numbers match Drupal 8 core compatibility, e.g.

So what should we have for Drupal 9?

Reply:
@vasike 
I think it should be obvious what branch should be for d9.2
https://github.com/drunomics/service-utils/tree/9.1.x#versions

Reply:
@mostepaniukvm  / @fago updated the base branch and also the drupal version

the question is 9.1.x is the right one or do we need a new one ... 9.2.x ?